daprdocs/content/en/java-sdk-docs/spring-boot/_index.md

```diff
@@ -323,6 +323,146 @@ daprWorkflowClient.raiseEvent(instanceId, "MyEvenet", event);
 
 Check the [Dapr Workflow documentation](https://docs.dapr.io/developing-applications/building-blocks/workflow/workflow-overview/) for more information about how to work with Dapr Workflows.
 
+## Using Dapr Cloud Config with Spring Boot
+
+To enable dapr cloud config, you should add following properties in your application's config (properties for example):
+
+```properties
+# default enable is true, don't need to specify
+dapr.cloudconfig.enabled = true
+spring.config.import[0] = <schema>
+spring.config.import[1] = <schema>
+spring.config.import[2] = <schema>
+#... keep going if you want to import more configs
+```
+
+There are other config of the dapr cloud config, listed below:
+
+```properties
+#enable dapr cloud config or not (default = true).
+dapr.cloudconfig.enabled=true
+#timeout for getting dapr config (include wait for dapr sidecar) (default = 2000).
+dapr.cloudconfig.timeout=2000
+#whether enable dapr client wait for sidecar, if no response, will throw IOException (default = false).
+dapr.cloudconfig.wait-sidecar-enabled=false
+#retries of dapr client wait for sidecar (default = 3).
+dapr.cloudconfig.wait-sidecar-retries=3
+```
+
+In Dapr Cloud Config component, we support two ways to imprt config: Secret Store API and Configuration API.
```

Comment:
@lony2003 typo on import

Reply:
I think I have listed below. Does it needs to be clearify at this place?